src/Microsoft.Management.Deployment/Converters.cpp

```diff
@@ -199,6 +199,8 @@ namespace winrt::Microsoft::Management::Deployment::implementation
             resultStatus = winrt::Microsoft::Management::Deployment::InstallResultStatus::NoApplicableInstallers;
             break;
         case APPINSTALLER_CLI_ERROR_UPDATE_NOT_APPLICABLE:
+        case APPINSTALLER_CLI_ERROR_UPGRADE_VERSION_UNKNOWN:
+        case APPINSTALLER_CLI_ERROR_UPGRADE_VERSION_NOT_NEWER:

```

Comment:
I'm not sure what the existing interactions look like, but we might need to separate these into new `InstallResultStatus` values as well.

It looks like we didn't split the "expected return codes" out in the `InstallResultStatus`, which itself seems like it might be an issue.